File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/checkout/OrderNotice.java

package com.artmurka.artmurkaapp.data.model.pojo.itemlist.checkout;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class OrderNotice {

    @SerializedName("notice_err")
    @Expose
    private String noticeErr;
    @SerializedName("notice")
    @Expose
    private String notice;

    public String getNoticeErr() {
        return noticeErr;
    }

    public void setNoticeErr(String noticeErr) {
        this.noticeErr = noticeErr;
    }

    public String getNotice() {
        return notice;
    }

    public void setNotice(String notice) {
        this.notice = notice;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/checkout/OrderNotice.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.checkout

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class OrderNotice {

    @SerializedName("notice_err")
    @Expose
    var noticeErr: String? = null
    @SerializedName("notice")
    @Expose
    var notice: String? = null

}
